### PR TITLE
Simplify Section 3.2.2

### LINKED ISSUE #94: Define "Authorization Domain Name" & use it or "FQDN" instead of "domain"
In CP Section 3.2.2.4.7, we use the term "Authorization Domain Name" but haven't defined it in the glossary, which we should do for clarity.

With that being done, it would be much more precise to use "Authorization Domain Name" or "FQDN" (as appropriate, keeping wildcards in mind) instead of "domain" in most cases throughout the CPS, especially in Section 3 (for example, 3.2.2).

### PR DESCRIPTION
**Pull request by aarongable**
Fixes #94

### CODE REVIEW

File: CPS.md [hunk 1]
```diff
@@ -285,19 +285,15 @@ Applicants are required to prove possession of the Private Key corresponding to
 
 ### 3.2.2 Authentication of organization and domain identity
 
-ISRG only issues Domain Validation (DV) certificates. When a certificate request includes a list of FQDNs in a SAN list, all domains in the list are fully validated prior to issuance.
+ISRG only issues Domain Validation (DV) certificates. All FQDNs listed in the certificate request are fully validated prior to issuance.
```
Comment (bdaehlie) [inline CPS.md:288]: Is it possible for a certificate request to somehow include a domain that would not be validated? Like, can an FQDN be included in any other field in a request? Seems like we should constrain this statement somehow?

Comment (aarongable) [reply CPS.md:288]: We could say "All FQDNs which will be listed in the certificate are fully...", maybe? That most closely matches the language from the BRs:
> The CA SHALL confirm that prior to issuance, the CA has validated each Fully‐Qualified Domain Name (FQDN) listed in the Certificate as follows:

Comment (bdaehlie) [reply CPS.md:288]: I like your suggestion. Let's do that.

Comment (aarongable) [reply CPS.md]: Done!

**Pull request merged**